src/components/Camera/style.css

```diff
@@ -36,42 +42,76 @@
   }
 }
 
+.fullScreenCenteredButton {
+  padding: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
+
 .btn:extend(.btn, .btn-centered, .btn-primary) {
   @media (--small-viewport) {
-    padding: 0;
-    position: absolute;
-    bottom: 48px;
-    margin-left: auto;
-    margin-right: auto;
-    left: 0;
-    right: 0;
+    .fullScreenCenteredButton();
   }
-  .btnText {
-    @media (--small-viewport) {
-      display: none;
-    }
+
+  .livenessCamera & {
+    .fullScreenCenteredButton();
+  }
+}
+
+.recordingIndicator {
+  font-size: 11px;
+  text-align: center;
+
+  /* Red circle */
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 8px;
+    height: 8px;
+    margin-right: 5px;
+    background: #DC2A2A;
+    border-radius: 4px;
+    line-height: 120%;
   }
 }
 
+.fullScreenRoundedIcon() {
```

Comment:
this repetition can be avoided by doing:
```
.fullScreenStartRecordingIcon, . fullScreenStopRecordingIcon {
 (...)
}

.fullScreenStartRecordingIcon() {
   (...)
}

.fullScreenStopRecordingIcon() {
   (...)
}
```

Reply:
so this is not possible because a selector with media query is using these values as well